src/interaction.ts

```diff
@@ -18,92 +16,82 @@ import { signatureNoSighashType } from "unchained-bitcoin";
 /**
  * Constant representing a keystore which is unsupported due to the
  * kind of interaction or combination of paramters provided.
- *
- * @type {string}
  */
 export const UNSUPPORTED = "unsupported";
 
 /**
  * Constant representing a keystore pending activation by the user.
- *
- * @type {string}
  */
 export const PENDING = "pending";
 
 /**
  * Constant representing a keystore in active use.
- *
- * @type {string}
  */
 export const ACTIVE = "active";
 
 /**
  * Constant for messages at the "info" level.
- *
- * @type {string}
  */
 export const INFO = "info";
 
 /**
  * Constant for messages at the "warning" level.
- *
- * @type {string}
  */
 export const WARNING = "warning";
 
 /**
  * Constant for messages at the "error" level.
- *
- * @type {string}
  */
 export const ERROR = "error";
 
 /**
  * Enumeration of possible keystore states ([PENDING]{@link module:interaction.PENDING}|[ACTIVE]{@link module:interaction.ACTIVE}|[UNSUPPORTED]{@link module:interaction.UNSUPPORTED}).
  *
- * @constant
- * @enum {string}
- * @default
- *
  */
 export const STATES = [PENDING, ACTIVE, UNSUPPORTED];
 
 /**
  * Enumeration of possible message levels ([INFO]{@link module:interaction.INFO}|[WARNING]{@link module:interaction.WARNING}|[ERROR]{@link module:interaction.ERROR}).
- *
- * @constant
- * @enum {string}
- * @default
- *
  */
 export const LEVELS = [INFO, WARNING, ERROR];
 
 /**
- * Represents an image in a message returned by an interaction.
+ * Represents a message returned by an interaction.
  *
- * @typedef module:interaction.MessageImage
- * @type {Object}
- * @property {string} label - a human-readable label for the image
- * @property {string} mimeType - the MIME type of the image
- * @property {string} data - base64-encoded image data
+ * Message objects may have additional properties.
  */
+type Message = {
```

Comment:
actually if you've got this type you could export it and use it in the place where I commented on the `any` previously.

Reply:
I don't think it's correct to export a type just so a test can use it. The test should be emulating a caller against an API. It should only have access to what an outside caller would have.

Reply:
Found to be needed for an error which wasn't previously being thrown. Changed in 35668a8805582ecc715ddc5e5214ac4de91c22ff